spec/migrations/20170710132935_custom_button_to_classes_spec.rb

```diff
@@ -0,0 +1,86 @@
+require_migration
+
+describe CustomButtonToClasses do
+  context 'CustomButton' do
+    let(:params) { custom_button.options }
+    let!(:custom_button) { migration_stub(:CustomButton).create!(create_params) }
+
+    migration_context :up do
+      let(:create_params) { {:options => {:button_image => 1} } }
+
+      it 'sets the image class and color' do
+        migrate
+        custom_button.reload
+
+        expect(params).to have_key(:button_icon)
+        expect(params).not_to have_key(:button_image)
+      end
+
+      context 'button with default color' do
+        let(:create_params) { {:options => {:button_image => 6} } }
+        it 'sets the image class only' do
+          migrate
+          custom_button.reload
+
+          expect(params).to have_key(:button_icon)
+          expect(params).not_to have_key(:button_color)
```

Comment:
I'd add `expect(params).not_to have_key(:button_image)` here.
So it'll be very obvious to readers the difference between the 2 cases. Will be helpful to readers like me.